test/regress/cli/regress1/strings/proj-issue659.smt2

```diff
@@ -0,0 +1,9 @@
+; COMMAND-LINE: -q
```

Comment:
why is this needed?

Reply:
Since the regression involves partial functions whose model can't be checked.

Reply:
So in that case maybe it is better to disable a tester rather than doing -q?

Reply:
No, it is better to use `q` since this still (partially) checks the model is correct, i.e. nothing is "false" but unevaluated is permitted.

Reply:
(FYI, `-q` is already used in 110 regressions)